debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/KafkaConnectRecordParser.java

```diff
@@ -189,9 +189,14 @@ protected void parseValueFields(Struct value, Record r) {
         Struct after = value.getStruct("after");
         // TODO: error handle before is NULL
         Struct before = value.getStruct("before");
+        if (r.op.equals("d") && before != null) {
+            // in case of delete, the before struct contains the values
+            after = before;
```

Comment:
No, for DELETE events `after` section is null and `before` section have the values for all the columns which we need for comparision.
We just wanted the before values in our queue events, so that we can compare it with the preceeding UPDATE event. So i just placed it in `after` struct so that it gets available to us.